cmd/context/options.go

```diff
@@ -86,6 +87,7 @@ func (o *ContextOptions) InitFromEnvVars() {
 			usedEnvVars = append(usedEnvVars, model.OktetoTokenEnvVar)
 		}
 		o.Token = envToken
+		o.InferredToken = true
```

Comment:
it is considered that having an expired token as an OKTETO_TOKEN variable must also be notified.